interface/portraits/portraits_highelffel.gfx

```diff
@@ -0,0 +1,385 @@
+# hair, mouth, chin, nose, eyes, cheek, ear, neck
+spriteTypes = {
+
+	### highelf MALE ####
+
+	# CULTURE_INDEX: 0 = clothes behind, 1 = headgear behind, 2 = clothes, 3 = headgear mid, 4 = clothes infront, 5 = headgear
+
+	# highelf Male
+	portraitType = {
+		name = "PORTRAIT_highelffelgfx_male"
```

Comment:
Better put portrait declaration in `portraits_highelf` since they share everything except eye colors.